lib/llm/src/http/service/service_v2.rs

```diff
@@ -281,7 +281,13 @@ impl HttpService {
             })?;
 
             axum::serve(listener, router)
-                .with_graceful_shutdown(observer.cancelled_owned())
+                .with_graceful_shutdown(async {
+                    observer.cancelled_owned().await;
+                    tracing::info!("HTTP server shutdown requested");
+                    // accepting requests for 5 more seconds, to allow incorrectly routed requests to arrive
+                    tokio::time::sleep(Duration::from_secs(5)).await;
```

Comment:
small nit: is it worth making this configurable?

Reply:
across both graceful shutdowns in this file? The other once is also 5s.